src/overlay/OverlayMetrics.cpp

```diff
@@ -13,6 +13,8 @@ OverlayMetrics::OverlayMetrics(Application& app)
           app.getMetrics().NewMeter({"overlay", "message", "read"}, "message"))
     , mMessageWrite(
           app.getMetrics().NewMeter({"overlay", "message", "write"}, "message"))
+    , mMessageDrop(
+          app.getMetrics().NewMeter({"overlay", "message", "drop"}, "message"))
```

Comment:
needs to be added to the metrics.md docs

Reply:
Fixed (and rebased to resolve conflict).